ui/src/components/inspectExamplesDemo/InspectExamples.js

```diff
@@ -7,7 +7,7 @@ import ModalVideo from 'react-modal-video';
 
 import './inspectExamples.css';
 import httpRequests from '../../helpers/httpRequests';
-import config from '../../helpers/config.json'
+//import config from '../../helpers/config.json'
```

Comment:
If something is not needed anymore, please remove, or add a further comment why this might be needed in the future.